scripts/operations/configuration/refresh_master_storage_rack_resiliency_config.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python3
+#
+# MIT License
+#
+# (C) Copyright 2025 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+
+"""
+refresh_master_storage_rack_resiliency_config.py
+
+This script refreshes the Rack Resiliency playbook layer
+(`rack_resiliency_for_mgmt_nodes.yml`) for all Master and Storage NCNs.
+
+This script modifies the CFS component status of master and storage NCNs to clear out
+the Rack Resiliency playbook entry. It also sets the error count for the component to 0.
+This will cause CFS batcher to schedule new CFS sessions to run that playbook
+on the master and storage nodes.
+
+It checks if any Master or Storage NCN has the Rack Resiliency layer configured
+in CFS. If at least one node in either category has the layer, it proceeds to
```

Comment:
This wording indicates that the script will work if just a single master node has the layer, and no storage nodes do. Or vice versa. But looking at the code, that does not appear to be true. Either the code or the text should be modified, so they are in agreement.